tools/common/StaticRegistrar.cs

```diff
@@ -642,6 +642,10 @@ protected override bool HasThisAttribute (MethodDefinition method)
 			
 		protected override bool Is64Bits {
 			get {
+#if !MTOUCH
```

Comment:
Instead of adding this condition, you can remove the `IsSingleAssembly` check below (it's not true anymore that we can't emit size-specific code in single assembly mode).